clang/test/Driver/android-ndk-standalone.cpp

```diff
@@ -311,3 +311,19 @@
 // CHECK-X86_64-GCC: Found candidate GCC installation: {{.*}}i686-linux-android{{[/\\]}}4.9
 // CHECK-X86_64-GCC-NEXT: Found candidate GCC installation: {{.*}}x86_64-linux-android{{[/\\]}}4.9
 // CHECK-X86_64-GCC-NEXT: Selected GCC installation: {{.*}}x86_64-linux-android{{[/\\]}}4.9
+//
+// RUN: %clang -### %s 2>&1 \
+// RUN:     --target=riscv64-linux-android \
+// RUN:     -march=rv64i \
+// RUN:     --gcc-toolchain=%S/Inputs/basic_android_ndk_tree \
```

Comment:
Will defer to @DanAlbert.  I am not sure if basic_android_ndk_tree is relevant for this test - if so, we should add riscv64 components to that subdir.

Reply:
The `CHECK-ARM64` variant of the test has it. I think it's so the test can find the linker? As long as this test is essentially a copy of that with the target and arch names swapped out, it's at least not wronger than the existing tests.